ui/raidboss/data/triggers/shinryu-ex.js

```diff
@@ -152,62 +238,105 @@
+      regexDe: /:Hakkinryu starts using Todesurteil on (\y{Name})/,
       alertText: function(data, matches) {
         if (matches[1] == data.me)
-          return 'Death Sentence on YOU';
+          return {
+            en: 'Death Sentence on YOU',
+            de: 'Todesurteil auf DIR',
+          };
         if (data.role == 'healer')
-          return 'Death Sentence on ' + matches[1];
+          return {
+            en: 'Death Sentence on ' + matches[1],
+            de: 'Todesurteil auf ' + matches[1],
+          };
       },
       infoText: function(data, matches) {
         if (matches[1] != data.me && data.role == 'tank')
-          return 'Death Sentence on ' + matches[1];
+          return {
+            en: 'Death Sentence on ' + matches[1],
+            de: 'Todesurteil auf ' + matches[1],
+          };
       },
       tts: function(data) {
         if (data.role == 'healer' || data.role == 'tank')
-          return 'death sentence';
+          return {
+            en: 'Death Sentence',
+            de: 'Todesurteil',
+          };
       },
     },
     { id: 'ShinryuEx Tera Slash',
       regex: /:264B:Shinryu starts using Tera Slash on (\y{Name})/,
+      regexDe: /:264B:Shinryu starts using Tera-Schlag on (\y{Name})/,
       alertText: function(data, matches) {
         if (matches[1] == data.me)
-          return 'Tank Buster on YOU';
+          return {
+            en: 'Tank Buster on YOU',
+            de: 'Tankbuster auf DIR',
+          };
         if (data.role == 'tank')
-          return 'Tank Swap';
+          return {
+            en: 'Tank Swap',
+            de: 'Tankwechsel',
+          };
         if (data.role == 'healer')
-          return 'Tank Buster on ' + matches[1];
+          return {
+            en: 'Tank Buster on ' + matches[1],
+            de: 'Tankbuster auf ' + matches[1],
+          };
       },
       tts: function(data, matches) {
         if (matches[1] == data.me || data.role == 'healer')
-          return 'Tank Buster';
+          return {
+            en: 'Tank Buster,
+            de: 'Tenkbasta',
+          };
         if (data.role == 'tank')
-          return 'Tank Swap';
+          return {
+            en: 'Tank Swap',
+            de: 'Tenkwechsel',
+          };
       },
     },
     { id: 'ShinryuEx Wormwail',
       regex: /:Shinryu starts using Wormwail/,
-      alertText: function(data) { return 'be inside hitbox'; },
+      regexDe: /:Shinryu starts using Shinryus Ruf/,
+      alertText: function(data) { return {
+        en: 'be inside hitbox',
+        de: 'In seine Hitbox',
+      }; },
       tts: 'get inside',
     },
     { id: 'ShinryuEx Breath',
       regex: /:264A:Shinryu starts using Benighting Breath/,
-      alertText: function(data) { return 'front cleave'; },
+      regexDe: /:264A:Shinryu starts using Dunkelhauch/,
+      alertText: function(data) { return {
+        en: 'front cleave',
+        de: 'Frontalcleave',
+      }; },
       tts: 'cleave',
```

Comment:
I'm not sure if this is intentional, but I see a bunch of missed translations (like this cleave) in a number of these files.  Were you going to come back and get these too?